test_requirements.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
```

Comment:
That was by design to stop having to run the full install-dependencies phase every time you want to run the unit tests locally.
The readme should (though needs to be made consistent across repos) say:

```shell
make
make test
```